packages/admin-ui/components/HeaderBar.tsx

```diff
@@ -0,0 +1,90 @@
+import { useRouter } from 'next/router';
+import styled from 'styled-components';
+import { Button, ButtonProps, Layout, Space } from 'antd';
+import LogoBar from '@vulcan-sql/admin-ui/components/LogoBar';
+import SharePopover from '@vulcan-sql/admin-ui/components/SharePopover';
+
+const { Header } = Layout;
+
+enum Path {
+  Onboarding = '/setup',
+  Modeling = '/modeling',
+  Explore = '/explore',
+}
```

Comment:
Please help moving this to a `path.ts` in enum folder.

Reply:
Done by 7ef4121f